chromium_src/chrome/browser/ui/test/test_browser_ui.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/test/test_browser_ui.h"
+
+#include "third_party/googletest/src/googletest/include/gtest/gtest.h"
```

Comment:
`#include "testing/gtest/include/gtest/gtest.h"` should be used.

Reply:
Fixed in d796000bfc0b57e79de47722732e548bd911f558.